### PR TITLE
use the faster to install google-closure-compiler-js

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -269,7 +268,7 @@ process.nextTick(function () {
         fs.writeFileSync(fpath, code);
         var output;
         try {
-          output = "" + child_process.execSync('node_modules/closurecompiler/bin/ccjs ' +
+          output = "" + child_process.execSync('./node_modules/.bin/google-closure-compiler-js ' +
```
Comment (ljharb) [inline build.js:271]: we shouldn't ever be (or have been) hardcoding `node_modules` - let's use `which('google-closure-compiler-js')` and https://www.npmjs.com/package/npm-which instead?

**Review by ljharb — CHANGES_REQUESTED** [review]
Thanks, seems like a good change.

File: build.js (hunk 1 above)
Comment (graingert) [reply build.js:271]: require('npm-which')(__dirname) or process.cwd() ?

Comment (ljharb) [reply build.js:271]: `process.cwd()`, i think, since all commands should be run from project root (assuming both work)

Comment (graingert) [reply build.js:271]: everything in this file hardcodes node_modules, and I don't want to touch everything in this PR

Comment (ljharb) [reply build.js:271]: alright, fair enough - we can leave it to a separate PR

**Review by ljharb — APPROVED** [review]

**Review by chicoxyzzy — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
Thank you @graingert!

**Pull request merged**